src/v3/src/components/InformationalText/InformationalText.tsx

```diff
@@ -35,13 +35,15 @@ const InformationalText: UISchemaElementComponent<{
     content,
     dataSe,
     variant,
+    bottomMargin,
```

Comment:
Is allowing the UI Schema should control the margin size like this a good idea? I think that the components should have a standardized spacing so they always appear the same way. If the design has more spacing we should evaluate if the prescribed spacing in the mocks is critical or not.

Reply:
I just saw the other comment below about why this was added. If the spacing should be increased it would ideally be across the board for all `InformationalText` components rather than configurable via the UI Schema.

Reply:
Oh waittt, the tight spacing was because these description elements have noMargin: true on them. I don't think the bottomMargin is necessary, we can remove that and then also remove the noMargin 👍

Reply:
oh wow it was right there dont know how I missed that haha 🤦